### PR TITLE
Balances windows (and by extension some walls)

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Updates all maps to include more regular windows.
Increases rod cost of rglass windows from 1 to 5 rods (does this ened to be more?)
Adjusts mapping standards to reflect new window use. (Fixes and clarifies some things there too)

- [ ] Consider slightly buffing rglass and glass window health values.

Gateway is no longer reinforced as it is now only and always beach

Standardises arrival shuttle, the front fell off.
Arrivals shuttle is now indestructable, this will mean players have a safe refuge every time they spawn in. No more meteor riddled shuttles.
image

Updates mapping contribution guidelines to reflect these, bumped minimum acceptable SDMM version to 2.

## Why It's Good For The Game
Currently, we use rglass almost everywhere. Now that windows match walls in style it can look quite out of place when rglass is used in non-secure areas.
This means that all non-secure areas use regular glass.
Rglass windows are still used on:

- External (to space) windows
- Transitions between unsecure and secure boundaries where no easily-accessible secondary security mechanism is available
- Places where rwalls are used (generally matches above criteria)

Other changes have been made to remove rwalls from areas that don't need them, secure areas generally only have a "perimiter" of rwalls and the walls within are regular walls.

## Changeling
:cl:
tweak: Most reinforced glass window instances in non-secure areas replaced with regular glass windows
tweak: Reinforced glass now costs 5 rods to make, not 1
tweak: The arrivals shuttle is now make of indestructable materials to protect new spawns
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -73,7 +73,10 @@ GLOBAL_LIST_INIT(glass_recipes, list (
 		var/obj/item/stack/rods/V  = W
 		var/obj/item/stack/sheet/rglass/RG = new (user.loc)
 		RG.add_fingerprint(user)
-		V.use(1)
+		if(V.get_amount() < 5)
+			to_chat(user, "<b>There is not enough rods in this stack. You need 5 rods.</b>")
+			return
+		V.use(5)
```
Comment (Contrabang) [inline code/game/objects/items/stacks/sheets/glass.dm:79]:
```suggestion
		if(V.use(5))
			to_chat(user, "<b>There is not enough rods in this stack. You need 5 rods.</b>")
			return
```
I think this works too?

Comment (S34NW) [reply code/game/objects/items/stacks/sheets/glass.dm]: Thanks, yeah it does with the `!`

**Comment by Scribble-Sheep** [comment]
I'm all for this. While I think the new rwindows look decent enough on their own, I was kind of upset to see how little they vibed with the look of the chapel the other night, compared to the old sprites, so this fixes the only complaint I really had with them.

**Comment by Denthamos** [comment]
Not enough detail on where regular windows is replacing reinforced windows.  Need more pictures of departments, external areas of departments, and such where you are making these blanket changes instead of a single picture of just the arrivals shuttle.

This is potentially a big change that can quickly change the dynamic due to how easy it is to break, and correct me if I am wrong but does not regular glass windows have a faster deconstruction time compared to reinforced windows?

**Comment by S34NW** [comment]
> Not enough detail on where regular windows is replacing reinforced windows. Need more pictures of departments, external areas of departments, and such where you are making these blanket changes instead of a single picture of just the arrivals shuttle.
> 
> This is potentially a big change that can quickly change the dynamic due to how easy it is to break, and correct me if I am wrong but does not regular glass windows have a faster deconstruction time compared to reinforced windows?

I am not going to screenshot every single window change across all 3 maps when there is a diffbot for this exact purpose. It literally highlights where windows have been changed. I agree that screenshots for smaller projects or ones that touch certain areas is needed, but when you've essentially changed the whole station there is no point in replicating MDB manually.

**Comment by S34NW** [comment]
> 
> Also, I don't think we should control what version of SDMM a person uses to map unless there is a very good reason that I am missing here. I myself cannot map with SDMM 2.0 right now because it is very broken and unusable on AMD GPUs.

SDMM 1 if used now seems to cause mass variable shuffling on our maps, and the user will have to remake their map edits.

**Comment by S34NW** [comment]
The amount of effort needed to fight for this is too great, RIP.

**Pull request closed without merging**